src/component/application/application-view.jsx

```diff
@@ -89,18 +89,21 @@ function ApplicationView({
                 <hr />
                 <List>
                     {seenToggles.map(
-                        ({ name, description, enabled, notFound }, i) => (
+                        (
+                            { name, description, enabled, notFound, project },
+                            i
+                        ) => (
                             <ConditionallyRender
                                 key={`toggle_conditional_${name}`}
                                 condition={notFound}
                                 show={notFoundListItem({
-                                    createUrl: '/features/create',
+                                    createUrl: `/projects/${project}/toggles`,
                                     name,
                                     permission: CREATE_FEATURE,
                                     i,
                                 })}
                                 elseShow={foundListItem({
-                                    viewUrl: '/features/strategies',
+                                    viewUrl: `/projects/${project}/toggles/${name}/strategies`,
```

Comment:
we must agree on `toggles` vs `features`. Currently the API will use `features`. I find it weird if we use other urls inside the UI. 

I would prefer the ability to prefix the url wiht "/api/admin" and get the JSON representation.

Reply:
Redone to features.